### PR TITLE
[Nette] Added ContainerBuilder expand() Rectors

### PR DESCRIPTION
**Pull request by EdoBarnas**
1. Nette\DI\ContainerBuilder->expand('string') => Nette\DI\ContainerBuilder->parameters['string')
2. Nette\DI\ContainerBuilder->expand(object|array) => Nette\DI\Helpers::expand(object|array, Nette\DI\ContainerBuilder->parameters)

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Hey Eda, thanks for your first Rector PR!

I'll look on it later. At the moment, could you improve the name to describe specific Rector?
Something like `[Nette] Added ContainerBuilder extract() Rector`

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,6 @@
 /example
 /vendor
 composer.lock
+
+.idea
+.gitignore
```
Comment (TomasVotruba) [inline .gitignore]: This doesn't make much sense. What is the motivation for that?

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,6 @@
 /example
 /vendor
 composer.lock
+
+.idea
```
Comment (TomasVotruba) [inline .gitignore]: I recommend adding any IDE related files to **global gitignore**.

Here is how: https://stackoverflow.com/a/7335487/1348344

File: src/Testing/PHPUnit/AbstractRectorTestCase.php [hunk 3]
```diff
@@ -38,6 +39,11 @@ protected function doTestFileMatchesExpectedContent(string $file, string $recons
             $this->getRectorClasses()
         );
 
+//        if ($this->getRectorClasses()[0] === ExpandFunctionToStaticExpandFunctionRector::class) {
+//            print($reconstructedFileContent);
+//            exit;
+//        }
```
Comment (TomasVotruba) [inline src/Testing/PHPUnit/AbstractRectorTestCase.php]: Maybe some debug code

File: src/config/level/nette/nette24.yml [hunk 4]
```diff
@@ -5,6 +5,8 @@ rectors:
     Rector\Rector\Contrib\Nette\Utils\NetteObjectToSmartTraitRector: ~
     Rector\Rector\Contrib\Nette\Utils\MagicMethodRector: ~
     Rector\Rector\Contrib\Nette\DI\SetEntityToStatementRector: ~
+    Rector\Rector\Contrib\Nette\DI\ExpandFunctionToParametersArrayRector: ~
+    Rector\Rector\Contrib\Nette\DI\ExpandFunctionToStaticExpandFunctionRector: ~
```
Comment (TomasVotruba) [inline src/config/level/nette/nette24.yml]: Awesome!

File: tests/Rector/Contrib/Nette/DI/ExpandFunctionToParametersArrayRector/Test.php [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    /**
+     * @return string[]
+     */
+    protected function getRectorClasses(): array
+    {
+        return [ExpandFunctionToParametersArrayRector::class];
+    }
+}
```
Comment (TomasVotruba) [inline tests/Rector/Contrib/Nette/DI/ExpandFunctionToParametersArrayRector/Test.php]: Perfect! :clap:

File: src/NodeAnalyzer/MethodArgumentAnalyzer.php [hunk 6]
```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types=1);
+
+namespace Rector\NodeAnalyzer;
+
+use PhpParser\Node;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Scalar\String_;
+
+final class MethodArgumentAnalyzer
```
Comment (TomasVotruba) [inline src/NodeAnalyzer/MethodArgumentAnalyzer.php]: Nice helper :+1:

**Comment by TomasVotruba** [comment]
Very nice job. Feels like reading my own code, nice feeling actually :) 

It looks like there are few coding standard violations. I recommend running this before PR:

```
composer check-cs
```

...to show them. And to fix them:

```
composer fix-cs
```

**Comment by EdoBarnas** [comment]
Thanks you for the opportunity to be better

File: .gitignore (hunk 2 above)
Comment (TomasVotruba) [reply .gitignore]: Could remove this line please? It could motivate people to add all IDE's files instead of ignoring them globally, which would be mess.

**Pull request merged**